README.md

```diff
@@ -2,4 +2,6 @@
 a microservice online store to buy and sell
 
 * The poject's struct is showed as follow:
-![Architect](WebServiceDemo_V1.png)
\ No newline at end of file
+![Architect](WebServiceDemo_V1.png)
```

Comment:
why change this line?

Reply:
forgot to add newline at the end of file last time.